app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/UploadSupportingDocuments.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns
+
+import julienrf.json.derived
+import play.api.libs.json.{Json, OFormat}
+
+sealed trait UploadSupportingDocuments extends Product with Serializable
+
+object UploadSupportingDocuments {
+
+  final case class SupportingDocument(upscanReference: String)
```

Comment:
is this being used anywhere?

Reply:
no, not anymore, that should be taken out

Reply:
removed